docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,69 +1,177 @@
+create a secured, multi-node cluster with a connected {kib} instance. This
+option requires more manual steps, but results in a more robust cluster.
+
+[[run-docker-secure]]
+[discrete]
+=== Start {es} and {kib} in Docker
+Starting in {es} 8.0, security is enabled by default. With security enabled,
+{stack} {security-features} require TLS encryption for the transport networking
+layer, or your cluster will fail to start.
+
+If you're starting a single-node {es} cluster in a Docker container, security
+will be automatically enabled and configured for you. When you start {es} for
+the first time, the following security configuration occurs automatically:
+
+* {ref}/configuring-stack-security.html#stack-security-certificates[Certificates and keys] are generated
+for the transport and HTTP layers.
+* The Transport Layer Security (TLS) configuration settings are written to
+`elasticsearch.yml`.
+* A password is generated for the `elastic` user.
+* An enrollment token is generated for {kib}.
+
+You can then {kibana-ref}/docker.html[start {kib}] and enter the enrollment
+token, which is valid for 30 minutes. This token automatically applies the
+security settings from your {es} cluster, authenticates to {es} with the
+`kibana_system` user, and writes the security configuration to `kibana.yml`.
+
+[discrete]
+==== Prerequisites
+. Install the appropriate https://docs.docker.com/get-docker/[Docker application]
+for your operating system.
++
+[NOTE]
+====
+Make sure that Docker is allotted at least 4GiB of memory. In Docker Desktop,
+you configure resource usage on the Advanced tab in Preference (macOS) or
+Settings (Windows).
+====
+
+. Pull the {es} Docker image.
++
+--
+ifeval::["{release-state}"=="unreleased"]
 
-To get the default distributions of {es} and {kib} up and running in Docker,
-you can use Docker Compose.
+WARNING: Version {version} of {es} has not yet been released, so no
+Docker image is currently available for this version.
 
-. Create a `docker-compose.yml` file for the Elastic Stack.
-The following example brings up a three node cluster and Kibana so you can see how things work.
-This all-in-one configuration is a handy way to bring up your first dev cluster before
-you build a distributed deployment with multiple hosts.
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+
+endif::[]
+--
+
+. Pull the {kib} Docker image.
 +
+--
 ifeval::["{release-state}"=="unreleased"]
-NOTE: Version {version} of {es} has not been released,
-so the sample compose file is not yet available for this version.
-See the {stack-gs-current}/get-started-docker.html[current version] for the latest sample files.
+
+WARNING: Version {version} of {es} has not yet been released, so no
+Docker image is currently available for this version.
+
 endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
```

Comment:
Should we show the command anyway, like we do for Kibana in the following section?